pkgs/firehose/lib/src/github.dart

```diff
@@ -43,6 +43,20 @@ class Github {
 
   http.Client get httpClient => _httpClient ??= http.Client();
 
+  /// Write the given [markdownSummary] content to the GitHub
+  /// `GITHUB_STEP_SUMMARY` file. This will cause the markdown output to be
```

Comment:
not a HUGE issue, but if these features are documented somewhere, it'd be cool to include reference in the source – just so folks understand...